common/workunit/workunit.cpp

```diff
@@ -14240,17 +14240,17 @@ bool applyK8sYaml(const char *componentName, const char *wuid, const char *job,
         E->Release();
         return false;
     }
-    jobYaml.replaceString("%jobname", jobname.str());
+    jobYaml.replaceString("_HPCC_JOBNAME_", jobname.str());
```

Comment:
note to reviewer(s): was necessary to change our pseudo var substitution from leading '%' pattern, because as yaml, these cause invalid yaml issues.